Makefile

```diff
@@ -1,2 +1,9 @@
+.PHONY: build
+
 build:
-	- gcc -framework Carbon get_window.c
+#	- gcc -framework Carbon get_window.c
```

Comment:
Something old we did when trying to get it running on macOS, not needed anymore.